prow/cmd/build/controller.go

```diff
@@ -57,6 +58,27 @@ const (
 	controllerName = "prow-build-crd"
 )
 
+type pjClient struct {
+	pjc       prowjobset.Interface
+	namespace string
+}
+
+func newPjClient(pjc prowjobset.Interface, namespace string) *pjClient {
```

Comment:
This is unexported, the calling code should be able to just make a struct literal and you do not need the constructor

Reply:
This is removed now.